requirements.txt

```diff
@@ -11,9 +11,10 @@
 -e packages/greedy_infomax
 -e packages/self_supervised_learning
 -e packages/continual_learning
+-e packages/spatial_pooler
 
 # Install projects extra dependency
 # See `setup.cfg` for available extras.
 # -e .[imagenet]
 # -e .[gsc]
-# -e .[transformers]
+# -e .[transformers]
```

Comment:
Is this correct?

Reply:
I removed the whitespace ending. GitHub I think shows it as I removed a line and then added it back.